src/test/benchmarks/jetton/results.json

```diff
@@ -100,18 +100,27 @@
       "commit": null
     },
     {
-      "label": "1.5.3 with address functions in Tact",
+      "label": "1.5.3 with address functions in Tact (#1766)",
       "gas": {
         "transfer": "19944",
         "burn": "15565",
         "discovery": "9613"
       },
       "commit": "https://github.com/tact-lang/tact/pull/1766/"
     },
+    {
+      "label": "1.5.3 with deploy instead of send (#1832)",
```

Comment:
let's not duplicate this info and just keep it in the commit field

Reply:
this whole entry should go after the "1.5.3 without double checking of message opcodes" change since we are going to merge it later